interop/firefox-version.json

```diff
@@ -1,7 +1,16 @@
 {
-  "id": "firefox-js-libp2p-head",
-  "containerImageID": "firefox-js-libp2p-head",
+  "id": "chromium-js-libp2p-head",
```

Comment:
Why is `chromium-js-libp2p-head` in `interop/firefox-version.json`?

Reply:
copy pasta error 😓